backend/src/main/java/com/huellapositiva/infrastructure/orm/model/Credential.java

```diff
@@ -40,4 +38,16 @@
     @OneToOne(optional = false, cascade = CascadeType.ALL)
     @JoinColumn(name = "email_confirmation_id")
     private EmailConfirmation emailConfirmation;
+
+    @Override
```

Comment:
@ToString annotation allows you to exclude fields and will autogenerate the method for you